src/main/java/zmaster587/advancedRocketry/client/render/planet/RenderPlanetarySky.java

```diff
@@ -796,9 +797,11 @@ public static void renderPlanetPubHelper(Tessellator tessellator1, ResourceLocat
 
 	private void drawStar(Tessellator buffer, int solarOrbitalDistance, float sunSize, float r, float g, float b, float multiplier) {
 		//Set sun color and distance
+		IPlanetaryProvider planetaryProvider = (IPlanetaryProvider)mc.theWorld.provider;
+		DimensionProperties properties = (DimensionProperties)planetaryProvider.getDimensionProperties((int)mc.thePlayer.posX, (int)mc.thePlayer.posZ);
 		GL11.glColor4f(r, g , b ,Math.min((multiplier)*2f,1f));
 		buffer.startDrawingQuads();
-		float f10 = sunSize*30f*(202-solarOrbitalDistance)/100f;
+		float f10 = sunSize*20f*AstronomicalBodyHelper.getBodySizeMultiplier(solarOrbitalDistance);
```

Comment:
is there a particular reason the magic constant was changed from 30 to 20?

Reply:
I felt the sun looked really large. It doesn't need to be 20, I just thought it would look better.

Reply:
fair enough